openfisca_core/taxbenefitsystems.py

```diff
@@ -161,10 +189,16 @@ def add_variables_from_directory(self, directory):
             self.add_variables_from_directory(subdirectory)
 
     def add_variables(self, *variables):
+        """
+        Does the same than :any:`add_variable`, but can take several variables.
```

Comment:
I would put the whole sentence:
```
Adds a list of OpenFisca `Variable`s to the `TaxBenefitSystem`.

See also: `add_variable`
```

I'm not sure of the syntax for "see also" to be a reference